Dockerfile

```diff
@@ -0,0 +1,29 @@
+FROM python:3
+
+# Install numpy using system package manager
+RUN apt-get -y update && apt-get -y install libav-tools imagemagick libopencv-dev python-opencv
+
+# Install some special fonts we use in testing, etc..
+RUN apt-get -y install fonts-liberation
+
+RUN apt-get install -y locales && \
+    locale-gen C.UTF-8 && \
+    /usr/sbin/update-locale LANG=C.UTF-8
+
+ENV LC_ALL C.UTF-8
+
+# do we need all of these, maybe remove some of them?
+RUN pip install imageio numpy scipy matplotlib pandas sympy nose decorator tqdm pillow pytest
+
+# install scikit-image after the other deps, it doesn't cause errors this way.
+RUN pip install scikit-image sklearn
+
+# install from github since there are bugs in latest versions.
+RUN git clone https://github.com/Zulko/moviepy.git /var/src/moviepy
+RUN cd /var/src/moviepy/ && python setup.py install
+
+# install ffmpeg from imageio.
+RUN python -c "import imageio; imageio.plugins.ffmpeg.download()"
```

Comment:
This will install ffmpeg into a weird location not in the path, should we add a symlink or something to put in in a location that is more common?